speechmatics/cli.py

```diff
@@ -353,6 +353,8 @@ def get_transcription_config(
         types = None
         if audio_events_config and audio_events_config.get("types"):
             types = audio_events_config.get("types")
+        if args.get("event_types"):
+            types = str(args.get("event_types")).split(",")
```

Comment:
Maybe call this event_types rather than just types, to make it a bit more explicit. Types feels too close to a lot of keywords in programming